MONOREPO_ROADMAP.md

```diff
@@ -0,0 +1,383 @@
+> For example, will it infer the Y column of a chart if one is not provided?
+
+> [!NOTE]
+> Some thought should be given to a similar pattern with mapping logic
+
+Responsible for encapsulating Evidence's echarts configurations in a re-usable package. This package is extracted to enable Evidence charts to be rendered in NodeJS applications.
+
+This functionality was previously provided by `@evidence-dev/core-components` and `@evidence-dev/component-utilities`
+
+##### Does:
+
+- Abstract echarts interactions away from Svelte (e.g. makes [`Chart.svelte`](./packages/ui/core-components/src/lib/unsorted/viz/core/_Chart.svelte))
+- Apply echarts theme + styling (incl. user overrides)
+
+##### Does Not:
+- Depend on Svelte, React or any other framework
+- Validate data (e.g. column existence)
+- Summarize data
+
+
+
+#### `@evidence-dev/ui-svelte`
+
+Responsible for providing Visualization and UI Components for building
+[Evidence Apps](#evidence-application) and [Evidence Projects](#evidence-project)
+
+This functionality was previously provided by `@evidence-dev/core-components` and `@evidence-dev/component-utilities`
+
+##### Does:
+- Provide the components needed to build a quality, comprehensive data application
+
+##### Does Not:
+- Provide the building blocks for Application shells
+
+
+#### `@evidence-dev/icons`
+
+Responsible for supplementing existing icon libraries with Evidence-specific icons
+
+##### Does:
+- Provide icon sources using the [steeze-ui](https://github.com/steeze-ui/icons) format
+- Support themeing
+
+##### Does Not:
+- Depend on Svelte, React or any other framework
+
+#### `@evidence-dev/tailwind`
+
+Responsible for providing a consistent tailwind design language for all things Evidence
+
+##### Does:
+- Expose a tailwind preset
+- Load and apply [Evidence themes](#evidence-theme)
+
+
+### SDK Interface Packages
+
+#### `@evidence-dev/cli`
+
+Responsible for managing [Evidence Projects](#evidence-project) and [Datasources](#datasources) with an ergonomic, interactive TUI.
+
+This functionality was previously provided by `@evidence-dev/evidence`
+
+##### Does:
+- Provides terminal-based wizards for SDK workflows
+
+#### `create-evidence`
+
+Responsible for enabling `npm create evidence`
+
+##### Does:
+- Wrap the [cli](#evidence-devcli)
+
+##### Does Not:
+- Implement any functionality
+
+
+#### `@evidence-dev/vscode`
+
+Responsible for managing [Evidence Projects](#evidence-project) and [Datasources](#datasources) with a VS Code plugin.
+
+This functionality was previously provided by `evidence-vscode`
+
+##### Does:
+- Provides wizards for SDK workflows in VS Code
+
+---
+
+## Glossary
+
+### Evidence Terms
+
+#### Evidence the "Product"
+
+- BI as Code tool using Markdown + SQL to build fast, reactive reports and apps
+- Used by the Modern Data Professional
+- An [Application Shell](#application-shell) built with the [Evidence Framework](#evidence-the-framework)
+
+#### Evidence the "Framework"
```

Comment:
Can we just call this Evidence the SDK rather than adding a new term? Particularly framework which is loaded

Reply:
Per our discussion we are landing on `headless evidence`